debian/build.go

```diff
@@ -93,14 +94,23 @@ func (c cfg) run() error {
 		}
 	}
 
-	cmd := exec.Command("dpkg-buildpackage", "-us", "-uc", "-b")
+	cmd := exec.Command("dpkg-buildpackage", "-us", "-uc", "-b", "--host-arch=" + c.DebArch)
```

Comment:
since we do this twice, might be good to recreate the runCommand() function in this package.